build/AzurePipelinesTemplates/ProjectReunion-CreateNugetPackage-Job.yml

```diff
@@ -95,6 +95,13 @@ jobs:
       TargetFolder: '$(Build.ArtifactStagingDirectory)\AppXContents'
       flattenFolders: false
 
+  - powershell: |
+      $fullpackagePath = "${{ parameters.fullnupkgdir }}"
+      $targetsFilePath = "$(Build.SourcesDirectory)\build\NuSpecs"
+      Copy-Item -Path "$targetsFilePath\ProjectReunion-Nuget-Native.targets" -Destination "$fullpackagePath\build\native\Microsoft.ProjectReunion.Foundation.targets"
+      Copy-Item -Path "$targetsFilePath\ProjectReunion-Nuget-Common.targets" -Destination "$fullpackagePath\build\Microsoft.ProjectReunion.targets"
```

Comment:
Instead of doing this in powershell you could use the CopyFiles task, something like:

  - task: CopyFiles@2
    displayName: 'Copy Project Reunion full package targets'
    inputs:
      SourceFolder: '$(Build.SourcesDirectory)\build\NuSpecs'
      Contents: |
        ProjectReunion-Nuget-Native.targets
        ProjectReunion-Nuget-Common.targets
      TargetFolder: '${{ parameters.fullnupkgdir }}\build'
      flattenFolders: false

Reply:
The above is not equivalent. Those files go in different folders and need to be renamed.

---
In reply to: [520918811](https://github.com/microsoft/ProjectReunion/pull/284#discussion_r520918811) [](ancestors = 520918811)

Reply:
Is there any reason we don't check them into the repo with the names we want them to have?

Reply:
There is already a file with the name Microsoft.ProjectReunion.targets that is being used for a different purpose. We should probably do cleanup and remove it if it is not going to be used but that is out of the scope of this change. This change is mostly just to unblock me.

---
In reply to: [520943020](https://github.com/microsoft/ProjectReunion/pull/284#discussion_r520943020) [](ancestors = 520943020)

Reply:
ok - that makes sense for this change